src/ocrd/processor/builtin/dummy_processor.py

```diff
@@ -1,13 +1,15 @@
 # pylint: disable=missing-module-docstring,invalid-name
 from os.path import join, basename
-from typing import Optional
+from typing import Optional, Union
 
 import click
 
 from ocrd import Processor
 from ocrd.decorators import ocrd_cli_options, ocrd_cli_wrap_processor
 from ocrd.processor.ocrd_page_result import OcrdPageResult
+from ocrd_models.ocrd_file import ClientSideOcrdFile, OcrdFile
 from ocrd_models.ocrd_page import OcrdPage, to_xml
```

Comment:
BTW, IMO we should also export `OcrdPage` in ocrd_models directly.

Reply:
I've become weary of using `__init__.py` imports because it leads to circular imports eventually. But it is convenient, I'll test that.

Reply:
70453181 done